### DISCUSSION TITLE
Error in testcase? (Test-Open)

### DISCUSSION THREAD

**Discussion by chaewoojin** [ANLAB-KAIST/KENSv3#112] [Project Q&A 2. Transport Layer (TCP)] [answered]
testbind.cpp
![image](https://user-images.githubusercontent.com/89976473/194445865-f6f73439-5425-4ba3-9da6-3f6ff089be5c.png)
I wonder why the expected return value of second bind function is zero.
According to the comment above, it should fail, which is expected to return -1(or something other than 0).

**Comment by lagabulin** [comment, marked as answer]
I'm not a TA, but I guess EXPECT_NE means ret must not be 0. I thought you saw it as EXPECT_EQ.

**Reply by chaewoojin** [reply]
Oh!!!!! Thank you so much. I think you're right.
